src/components/ForestTypeDescription/lu/AssociationsTab.js

```diff
@@ -84,7 +84,7 @@ function AssociationsTab({ associationGroupCode }) {
                   <button
                     className={styles.link}
                     type="button"
-                    onClick={() => dispatch(setForestTypeInfo(type))}
+                    onClick={() => dispatch(setForestTypeDescription(type))}
```

Comment:
@friedjoff Something is wrong with the links to the associated groups in the associations tab. When a link is clicked it always shows the message about being in the wrong profile.

Reply:
Thanks, good catch, is fixed now.